frontend/src/styles/base/typography.scss

```diff
@@ -51,6 +51,11 @@
   background-color: transparent;
   border: 1px solid $darkBlue;
   border-radius: 3px;
+  transition: ease-in 0.4s opacity;
```

Comment:
the transition + opacity looks good. However, we are repeating the same code 4 times. We could create a scss (sass) mixin, call it  **defaultHoverState**, and add opacity on hover + transition there. Then just include the mixin in all 4 places.